### PR TITLE
Get single product client

### PR DESCRIPTION
**Pull request by Maxim-Zavidei**
# Overview:
* Implemented get product by id on client
* Css fix by @LilianBittar

### CODE REVIEW

File: src/Client/Pages/ProductDetails.razor [hunk 1]
```diff
@@ -5,7 +5,7 @@
 
 @if(product == null)
 {
-    <span>Loading Product...</span>
+    <span>@message</span>
```
Comment (LilianBittar) [inline src/Client/Pages/ProductDetails.razor:8]: Nice change :) keep up the good work.

**Review by LilianBittar — APPROVED** [review]
Gucci

**Comment by Maxim-Zavidei** [comment]
noiceee

**Comment by Maxim-Zavidei** [comment]
on icecream

**Pull request merged**